### PR TITLE
add lemon juice and fix bad sprite allocation

### PR DESCRIPTION
**Pull request by Shadowtheprotogen546**
## About the PR

added a lemon juice carton to the boozemat

## Why / Balance

Lemon juice now no longer requires botany to mass produce lemons

## Media

lemonjuice

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

Make sure to take this Changelog template out of the comment block in order for it to show up.
🆑 Floofi
- add: Added lemon juice to the Boozemat!

### CODE REVIEW

**Comment by UbaserB** [comment]
Why does the glass look like a lemon, this is weird

**Comment by Shadowtheprotogen546** [comment]
> Why does the glass look like a lemon, this is weird

Because that's what the lemon juice containers I buy look like

**Comment by Plykiya** [comment]
81uWodhYh3S

I don't mind the lemon shaped lemon juice container, but the sprite doesn't really look like a lemon to me... at least compared to our existing lemon sprite

**Comment by Shadowtheprotogen546** [comment]
> I don't mind the lemon shaped lemon juice container since I have those too, but the sprite doesn't really look like a lemon to me... at least compared to our existing lemon sprite

I don't want it to look to similar to the lime juice bottle and I don't want people to confuse it for an actual lemon.  This one is unique enough to net be seen as either.

**Comment by EmoGarbage404** [comment]
i think it looks a bit too much like a lemon?
images

perhaps go with a flatter bottom, ditch the leaf, and add an outline for more definition?

**Comment by Shadowtheprotogen546** [comment]
> perhaps go with a flatter bottom, ditch the leaf, and add an outline for more definition?

I like the leaf too much but added outline and changed the shape

**Comment by potato1234x** [comment]
imo just make it a yellow circle bottle

**Comment by Shadowtheprotogen546** [comment]
> imo just make it a yellow circle bottle

that wouldn't stand out

**Comment by Emisse** [comment]
Closed due to feature freeze May 10th-June 14th. Comment to have it reopen after this.

**Comment by Shadowtheprotogen546** [comment]
please reopen

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks-cartons.yml [hunk 1]
```diff
@@ -193,3 +193,18 @@
...
+  description: First it's sour, then it's still sour.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        reagents:
+        - ReagentId: JuiceLemon
+          Quantity: 50
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/lemonjuice.rsi
```
Comment (AJCM-git) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks-cartons.yml:210]: isnt this the same as DrinkLemonJuice but with a better name? Either move the description there and remove this one or make this one use a new sprite that its more like a carton and rename it to "lemon juice carton"

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks-cartons.yml (hunk 1 above)
Comment (Shadowtheprotogen546) [reply Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks-cartons.yml:210]: there was 2 lemon juice glasses.  with the metamorphic glass update, there wasn't a need for that anymore so I removed the useless one and replaced it with the lemon juice carton.  The carton is a seperate entity from the drink version of it, so the carton has a different description.  Now its just 2 entities.  The juice carton, and the juice when it's in a glass.

**Comment by Shadowtheprotogen546** [comment]
It has been 2 months since I posted this PR.  Can I please just get a yes or no so it can be closed?

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**